pkg/api/v1beta3/dynakube/properties.go

```diff
@@ -239,14 +239,17 @@ func (dk *DynaKube) ImagePullSecretReferences() []corev1.LocalObjectReference {
 }
 
 func (dk *DynaKube) NeedsReadOnlyOneAgents() bool {
-	return dk.HostMonitoringMode() || dk.CloudNativeFullstackMode()
```

Comment:
can you please add tests? because other properties have tests in `pkg/api/v1beta3/dynakube/properties_test.go`